snap/snapcraft.yaml

```diff
@@ -591,8 +593,8 @@ parts:
       - zip
 
   redis:
-    source: https://github.com/antirez/redis.git
-    source-tag: "6.0.8"
+    source: https://github.com/redis/redis.git
+    source-tag: "6.0.10"
```

Comment:
For Docker, the `hanoi` branch is using `6.0.9`. I assume we want these to be the same?
https://github.com/edgexfoundry/developer-scripts/blob/hanoi/compose-builder/.env#L33

Reply:
Sure, I can make that change...

Reply:
Just force pushed and adjusted the commit which bumped the snap to redis 6.0.10 to bump instead to 6.0.9.